server/database/index.js

```diff
@@ -87,9 +97,39 @@ function setResponse(response) {
  * @param req - expectation of a store name to be passed as storeName
  */
 app.get("/get-store", (req, res) => {
-    getStore(req.query.storeName);
-    res.json(jsonResponse);
-  });
+    var sql = "SELECT * FROM stores WHERE name = ?";
+    var query = connection.query(sql, [storeName], function (err, result) {
+        if (err) throw err;
+        res.json(result[0]);
+    });
+
+});
+
+/** Route to list all stores */
+app.get("/list-stores", (req, res) => {
+
+    var sql = "SELECT * FROM stores";
+    var query = connection.query(sql, function (err, result) {
+        if (err) throw err;
+        res.json(result);
+    });
+
+    //console.log(jsonResponse)
+    //res.json(jsonResponse);
+});
+
+/** Route to get store by ID
+ *  @param storeId - expects an integer value to be passed into query
+ */
+app.get("/get-store-id", (req, res) => {
```

Comment:
It seems like there are two handlers for this route? Is this meant to be or is this something to fix?